asllib/Native.ml

```diff
@@ -167,9 +167,14 @@ module NativeBackend = struct
   let read_from_bitvector positions bv =
     let positions = slices_to_positions as_int positions in
     let max_pos = List.fold_left int_max 1 positions in
```

Comment:
In `StaticInterpreter` the corresponding (?) line is `List.fold_left int_max 0 positions`. So `1` or `0` ? Or am I missing something?